Documentation/how-to-build-and-run-ilcompiler-in-console-shell-prompt.md

```diff
@@ -40,7 +37,7 @@ This will result in the following:
 This approach uses the same code-generator (RyuJIT), as [CoreCLR](https://github.com/dotnet/coreclr), for compiling the application. From the shell/command prompt, issue the following commands, from the folder containing your source file and project.json, to generate the native executable
 
     dotnet restore
-    dotnet compile --native --ilcpath c:\newilc
+    dotnet compile --native --ilcpath bin\Product\Windows_NT.x64.Debug\.nuget\publish1
```

Comment:
Currently we have .nuget inside Windows_NT.x64. Inside that folder publish1 is present.